src/modules/apiCall.js

```diff
@@ -0,0 +1,17 @@
+const dataJson = [''];
+const apiCall = async (homePageOption) => {
+  await fetch(`https://api.tvmaze.com/search/shows?q=${homePageOption[0]}`)
+    .then((data) => data.json())
+    .then((data) => {
+      dataJson.length = 0;
+      dataJson.push(data);
+    });
```

Comment:
- [ ] Please make sure that the comment popup is responsive look how it looks like when the user changes the screen size to mobile, I think you should use more responsive measures. 

| Yours  | original design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/74744787/211094981-028d9734-57b4-4f3a-a8e4-c898275d61cc.png)  | ![image](https://user-images.githubusercontent.com/74744787/211095042-21cfee70-9b58-46a3-81ab-dd99b5969655.png) |

Reply:
The comment popup is now responsive

Reply:
- [optional] Please I noticed that when a comment is submitted, I need to refresh the browser before the comment can be updated. Kindly work on this in order to enhance the user accessibility of your app. 👍